jobs/views.py

```diff
@@ -94,11 +94,15 @@ def create_job(request):
             subject = '{0} was submitted succesfully.'.format(new_job)
             message_plain = get_template(
             'jobs/email_templates/community_confirm.txt').render(
-                Context({ 'option': new_job })
+                Context({
+                    'option': new_job
+                })
```

Comment:
No need to split this across lines, just keep the final parenthesis on the next line